doc/managing-wallets.md

```diff
@@ -15,7 +15,7 @@ The following command, for example, creates a descriptor wallet. More informatio
 $ bitcoin-cli -named createwallet wallet_name="wallet-01" descriptors=true
 ```
 
-The `descriptors` parameter can be omitted if the intention is to create a legacy wallet. For now, the default type is the legacy wallet, but that is expected to change in a future release.
+The `descriptors` parameter can be set to false if the intention is to create a legacy wallet.
```

Comment:
If anyone wants to to this, they can read the docs. So maybe just drop this paragraph.

Reply:
@Sjors 

I tried to find information on creating a legacy wallet, but I couldn't easily find one other than this file.
If our goal is to discourage users from creating a new wallet as a legacy wallet, I think it makes to remove this line. However, if that's not our goal, I think this line shouldn't be removed.

This file is documentation on managing wallets. I don't think it's optimal not to mention how to create a legacy wallet in this file unless our goal is to make it challenging for the user to create a legacy wallet.